cmd/minikube/cmd/start.go

```diff
@@ -87,8 +86,9 @@ var startCmd = &cobra.Command{
 
 // Set default value for host data and config dir
 var (
-	hostConfigDirectory = "/var/lib/minishift/openshift.local.config"
-	hostDataDirectory   = "/var/lib/minishift/hostdata"
+	hostConfigDirectory  = "/var/lib/minishift/openshift.local.config"
+	hostDataDirectory    = "/var/lib/minishift/hostdata"
+	hostVolumesDirectory = "/var/lib/minishift/openshift.local.volumes"
```

Comment:
@hferentschik It is confirmed that `oc cluster up` does not create the directories. So I agree that we need to create the directories if do not exist.